CHANGELOG/v1.8.0-beta.1.md

```diff
@@ -0,0 +1,416 @@
+🚨 This is a BETA RELEASE. Use it only for testing purposes. If you find any bugs, file an [issue](https://github.com/kubernetes-sigs/cluster-api/issues/new).
+
+## Highlights
+* Add kubeadm v1beta4 types
```

Comment:
Just fyi. I will provide a highlights and deprecation section. Not sure if already for the beta tomorrow, but at least for the next releases